### PR TITLE
Remove broken link from bug-bounty page 

### LINKED ISSUE #1703: Remove broken link from bug bounty page
### Description
the link to the "submission form" added here: [Lisk Bug Bounty Program :: Lisk documentation](https://lisk.com/documentation/bug-bounty-program.html) doesn't work – as we don't have such a page on the website.
We have basically two options:

- Remove the link and leave only "[security@lisk.com](mailto:security@lisk.com)" as a contact option.
- Add a form somewhere on the new website.

Also, the text in the image is not visible in the dark theme, this should be fixed.

### Affected branch / URL
`main`, `development`

### Additional resources
Relevant resources to solve the issue, i.e. lips, code examples, blogs, videos, etc.

### PR DESCRIPTION
**Pull request by chris529**
Closes #1703

### CODE REVIEW

File: docs/modules/ROOT/pages/bug-bounty-program.adoc [hunk 1]
```diff
@@ -44,13 +44,14 @@ Serial vulnerabilities caused by the same underlying issue are treated as a sing
 
 The following table illustrated below lists the current remuneration structure for identifying bugs and vulnerabilities.
 
-image::intro/bug-bounty-payout1.png[ align="center"]
+// image::intro/bug-bounty-payout1.png[ align="center"]
+image::intro/bug-bounty-diagram.png[ align="center"]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc]: The label for **Medium** should be yellow, but it is green in the image.

File: docs/modules/ROOT/pages/bug-bounty-program.adoc [hunk 2]
```diff
@@ -44,13 +44,14 @@ Serial vulnerabilities caused by the same underlying issue are treated as a sing
 
 The following table illustrated below lists the current remuneration structure for identifying bugs and vulnerabilities.
 
-image::intro/bug-bounty-payout1.png[ align="center"]
+// image::intro/bug-bounty-payout1.png[ align="center"]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc]: There is a point in the issue description, which I am not sure about. @Tschakki Please elaborate.
> Add a form somewhere on the new website.

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/bug-bounty-program.adoc [hunk 3]
```diff
@@ -44,13 +44,14 @@ Serial vulnerabilities caused by the same underlying issue are treated as a sing
...
+image::intro/bug-bounty-diagram.png[ align="center"]
 
 === Report your Bug
 
 image::intro/bug-reportform1.png[role="related thumb right",200]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc:52]: What is CHF? and should CHF be LSK?

File: docs/modules/ROOT/pages/bug-bounty-program.adoc (hunk 1 above)
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc]: What is **CHF**? and should **CHF** be **LSK**?

Comment (chris529) [reply docs/modules/ROOT/pages/bug-bounty-program.adoc]: This is correct as the payouts are in Swiss Francs, and the international currency symbol for the Swiss Franc is CHF.

File: docs/modules/ROOT/pages/bug-bounty-program.adoc (hunk 2 above)
Comment (chris529) [reply docs/modules/ROOT/pages/bug-bounty-program.adoc]: Ah, this was choice between either adding a new report  form or requesting an email. imho, I think it is more preferable for the user to be able to describe the vulnerabilities/bugs found in his own words.

File: docs/modules/ROOT/pages/bug-bounty-program.adoc (hunk 1 above)
Comment (chris529) [reply docs/modules/ROOT/pages/bug-bounty-program.adoc]: Good point Talha, I will go back to Rita and ask her to update this 👍

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 🎉

File: docs/modules/ROOT/pages/bug-bounty-program.adoc [hunk 4]
```diff
@@ -45,7 +45,7 @@ Serial vulnerabilities caused by the same underlying issue are treated as a sing
 The following table illustrated below lists the current remuneration structure for identifying bugs and vulnerabilities.
 
 // image::intro/bug-bounty-payout1.png[ align="center"]
-image::intro/bug-bounty-diagram.png[ align="center"]
+image::intro/bug-bounty-diagram1.png[ align="center"]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc]: I would replace the existing diagram (bug-bounty-diagram) and not add the new diagram file (bug-bounty-diagram1) as this file may have been referenced on other pages as well.

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/bug-bounty-program.adoc (hunk 4 above)
Comment (chris529) [reply docs/modules/ROOT/pages/bug-bounty-program.adoc]: Now done 👍

**Review by TalhaMaliktz — APPROVED** [review]
Thanks, LGTM 🎉

**Pull request closed without merging**